src/app/_blocks/HubContentGrid/index.tsx

```diff
@@ -25,6 +26,12 @@ export default function HubContentGrid({ articles }: HubContentGridProps) {
 
   const [activeButton, setActiveButton] = useState<keyof ContentTypeArrays | 'All'>('All')
 
+  let windowWidth = window.innerWidth
+
+  window.addEventListener('resize', () => {
```

Comment:
I don't think this is necessary (because of the other comment about media queries), but anyway, event listeners in React need some extra care because they need to be cleaned up when the component unmounts. The best practice looks like this: https://www.pluralsight.com/resources/blog/guides/event-listeners-in-react-components

Reply:
thanks. i'll update it and shift to using pure css

Reply:
fixed in: https://github.com/subvisual/content-sub/pull/43/commits/f4f6c9e3320f47fbfd4c7a75a98bfccb58cdaf75